hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/Call.java

```diff
@@ -64,6 +67,13 @@ class Call {
   Call(int id, final Descriptors.MethodDescriptor md, Message param, final CellScanner cells,
     final Message responseDefaultType, int timeout, int priority, RpcCallback<Call> callback,
     MetricsConnection.CallStats callStats) {
+    this(id, md, param, cells, responseDefaultType, timeout, priority, Collections.emptyMap(),
```

Comment:
there are only 2 extra test-related usages of this constructor.. if there were many, i'd say this is good to avoid unnecessary changes. but with just 2 i might prefer keeping the one constructor so as not to create an unnecessary precedent.